pandas/tests/extension/test_numpy.py

```diff
@@ -330,6 +330,17 @@ def test_fillna_frame(self, data_missing):
         # Non-scalar "scalar" values.
         super().test_fillna_frame(data_missing)
 
+    def test_fillna_fill_other(self, data_missing):
```

Comment:
#40758 removed this test, which then started failing on this PR, so this restores it